sdk/eventhub/eventhubs-checkpointstore-blob/samples/v1/typescript/README.md

```diff
@@ -0,0 +1,82 @@
+| ----------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [createCustomPipeline.ts][createcustompipeline]                               | Demonstrates how to create a custom `Pipeline` that the Storage Blob Container Client uses to change the API version used when communicating with the service. //@@TS-MAGIC-NEWLINE@@ This may be useful if the environment you are targetting supports an older version of Storage Blob than is officially supported by the                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
This TS-MAGIC-NEWLINE stuff is supposed to be stripped out, and I suspect it's related to the way the comments are formatted. These summaries are also bordering on too long; maybe shorter summaries could be more useful at a glance in the table.

Reply:
Ah, I was hoping there was a way that everything after the newline would be stripped from the table. Alas, I'll redo my summaries.

Reply:
Oh, ha, I thought this was my event hubs PR. Nevermind :smile:

Reply:
It can be your PR @chradek 
I am willing to sell it you at no cost :)

Reply:
I kind of walked right into that one...

Reply:
yes, yes you did

Reply:
Fixed.
@chradek I have tweaked the text on the summary as well. Please take a look